cob_moveit_config/package.xml

```diff
@@ -0,0 +1,18 @@
+<package format="2">
+  <name>cob_moveit_config</name>
+  <version>0.6.6</version>
+  <description>
+     MoveIt config files for all cob and raw
+  </description>
+  <author email="mdl@ipa.fhg.de">Mathias Luedtke</author>
+  <maintainer email="mdl@ipa.fhg.de">Mathias Luedtke</maintainer>
+  <maintainer email="fxm@ipa.fhg.de">Felix Messmer</maintainer>
+
+  <license>BSD</license>
+
+  <buildtool_depend>catkin</buildtool_depend>
+
+  <exec_depend>moveit_setup_assistant</exec_depend>
```

Comment:
done in https://github.com/ipa320/cob_robots/pull/573/commits/4c9013ca2914748a9082036f83db016c4e6c1064 and https://github.com/ipa320/cob_manipulation/pull/91/commits/5d4c095891612ca137be2f357f4f66b678d81623